src/board/generic_esp32/generic-esp32-board.c

```diff
@@ -39,16 +46,28 @@ typedef struct spi_s {
 static spi_c lora_spi;
 
 static const char *TAG = "ESP32Board";
-
 static portMUX_TYPE my_spinlock = portMUX_INITIALIZER_UNLOCKED;
 
+static esp_timer_handle_t irq_timer_handle;
```

Comment:
this is a timer for measuring the irq?

Reply:
It's used to call the stack's internal `TimerIRQHandler()` function upon timer expiration.

Reply:
variable name is a little confusing. 'timer_handler' maybe? its unclear where its coming from, internal vs. external etc.